src/platformdirs/windows.py

```diff
@@ -24,7 +24,14 @@ def user_data_dir(self) -> str:
          ``%USERPROFILE%\\AppData\\Roaming\\$appauthor\\$appname`` (roaming)
         """
         const = "CSIDL_APPDATA" if self.roaming else "CSIDL_LOCAL_APPDATA"
-        path = os.path.normpath(get_win_folder(const))
+        path = get_win_folder(const)
```

Comment:
Seems unrelated change.

Reply:
I had to add support for a None value in get_win_folder() since Documents has no environment variable. Since, it otherwise would have raised an error here, I agree that this wasn't needed. Should I implement the default values for folders besides the documents folder in a separate PR?

Reply:
Yes 🙏